cron_tasks.go

```diff
@@ -0,0 +1,29 @@
+package scalingo
+
+import (
+	"gopkg.in/errgo.v1"
+)
+
+type CronTasksService interface {
+	CronTasksAdd(app string, params CronTasks) (CronTasks, error)
+}
+
+var _ CronTasksService = (*Client)(nil)
+
+type Job struct {
+	Command string `json:"command"`
+	Size    string `json:"size"`
+}
+
+type CronTasks struct {
+	Jobs []Job `json:"jobs"`
+}
+
+func (c *Client) CronTasksAdd(app string, params CronTasks) (CronTasks, error) {
+	resp := CronTasks{}
+	err := c.ScalingoAPI().SubresourceAdd("apps", app, "cron_tasks", params, &resp)
+	if err != nil {
+		return CronTasks{}, errgo.Mask(err)
```

Comment:
Sorry I didn't noticed it during my first review but we don't use `errgo.Mask` since a long time. The rule is the following. If this is a legacy project which still uses `errgo`, please use `errgo.Notef`. If the project uses `errors`, please use `errors.Wrap`.

In this case, the project is legacy and use `errgo`